### PR TITLE
Add raster metrics visualizer to performance page

### PR DESCRIPTION
**Pull request by kenzieschmoll**
raster-metrics-small-gif

We should still iterate on this UI with UXR (@InMatrix @ayanogawa). The goal is to get something landed here that we can gather initial feedback from and iterate upon. This feature is hidden behind a flag, so will not be enabled by default.

@iskakaushik @jacob314

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/performance/raster_metrics.dart [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+        ),
+      ),
+      actions: [
+        DialogCloseButton(),
+      ],
+    );
+  }
+}
+
+class RasterMetricsController {
```
Comment (elliette) [inline packages/devtools_app/lib/src/screens/performance/raster_metrics.dart:236]: This is just a curiosity - I know in other places we give the controller its own file. Is there a reason to include it here and not separate it out?

**Review by elliette — APPROVED** [review]

File: packages/devtools_app/lib/src/screens/performance/raster_metrics.dart (hunk 1 above)
Comment (kenzieschmoll) [reply packages/devtools_app/lib/src/screens/performance/raster_metrics.dart:236]: I'll move it to a separate file in a follow up PR.

**Comment by zanderso** [comment]
+1 great work!

**Comment by InMatrix** [comment]
How to turn on the flag? I look forward to trying this out.

**Pull request merged**